2022-06-07 20:29:53,585 INFO    MainThread:13803 [wandb_setup.py:_flush():76] Loading settings from /home/shuffle/.config/wandb/settings
2022-06-07 20:29:53,586 INFO    MainThread:13803 [wandb_setup.py:_flush():76] Loading settings from /home/shuffle/DL-IAD-HSE/DeepLearning/Метод обратного распространения ошибки. Свёртки и свёрточные слои/Seminar/wandb/settings
2022-06-07 20:29:53,586 INFO    MainThread:13803 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-06-07 20:29:53,586 INFO    MainThread:13803 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-06-07 20:29:53,586 WARNING MainThread:13803 [wandb_setup.py:_flush():76] interrupted
Traceback (most recent call last):
  File "/home/shuffle/anaconda3/lib/python3.9/site-packages/IPython/core/interactiveshell.py", line 3369, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "/tmp/ipykernel_13803/4016653966.py", line 4, in <cell line: 4>
    wandb.init(project="sem-02")
  File "/home/shuffle/anaconda3/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1021, in init
    raise e
  File "/home/shuffle/anaconda3/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 996, in init
    wi.setup(kwargs)
  File "/home/shuffle/anaconda3/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 237, in setup
    wandb_login._login(
  File "/home/shuffle/anaconda3/lib/python3.9/site-packages/wandb/sdk/wandb_login.py", line 297, in _login
    wlogin.prompt_api_key()
  File "/home/shuffle/anaconda3/lib/python3.9/site-packages/wandb/sdk/wandb_login.py", line 220, in prompt_api_key
    key, status = self._prompt_api_key()
  File "/home/shuffle/anaconda3/lib/python3.9/site-packages/wandb/sdk/wandb_login.py", line 200, in _prompt_api_key
    key = apikey.prompt_api_key(
  File "/home/shuffle/anaconda3/lib/python3.9/site-packages/wandb/sdk/lib/apikey.py", line 120, in prompt_api_key
    key = input_callback(api_ask).strip()
  File "/home/shuffle/anaconda3/lib/python3.9/site-packages/ipykernel/kernelbase.py", line 1057, in getpass
    return self._input_request(
  File "/home/shuffle/anaconda3/lib/python3.9/site-packages/ipykernel/kernelbase.py", line 1120, in _input_request
    raise KeyboardInterrupt("Interrupted by user") from None
KeyboardInterrupt: Interrupted by user
2022-06-07 20:29:53,590 INFO    MainThread:13803 [wandb_init.py:_log_setup():428] Logging user logs to /home/shuffle/DL-IAD-HSE/DeepLearning/Метод обратного распространения ошибки. Свёртки и свёрточные слои/Seminar/wandb/run-20220607_202953-1t9gr6dk/logs/debug.log
2022-06-07 20:29:53,590 INFO    MainThread:13803 [wandb_init.py:_log_setup():429] Logging internal logs to /home/shuffle/DL-IAD-HSE/DeepLearning/Метод обратного распространения ошибки. Свёртки и свёрточные слои/Seminar/wandb/run-20220607_202953-1t9gr6dk/logs/debug-internal.log
2022-06-07 20:29:53,590 INFO    MainThread:13803 [wandb_init.py:_jupyter_setup():378] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f4a78ca8cd0>
2022-06-07 20:29:53,591 INFO    MainThread:13803 [wandb_init.py:init():462] calling init triggers
2022-06-07 20:29:53,591 INFO    MainThread:13803 [wandb_init.py:init():465] wandb.init called with sweep_config: {}
config: {}
2022-06-07 20:29:53,591 INFO    MainThread:13803 [wandb_init.py:init():515] starting backend
2022-06-07 20:29:53,592 INFO    MainThread:13803 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-06-07 20:29:53,613 INFO    MainThread:13803 [backend.py:ensure_launched():217] starting backend process...
2022-06-07 20:29:53,626 INFO    MainThread:13803 [backend.py:ensure_launched():222] started backend process with pid: 31504
2022-06-07 20:29:53,628 INFO    MainThread:13803 [wandb_init.py:init():525] backend started and connected
2022-06-07 20:29:53,665 INFO    MainThread:13803 [wandb_run.py:_label_probe_notebook():1089] probe notebook
2022-06-07 20:29:53,666 INFO    MainThread:13803 [wandb_run.py:_label_probe_notebook():1099] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-06-07 20:29:53,666 INFO    MainThread:13803 [wandb_init.py:init():596] updated telemetry
2022-06-07 20:29:53,694 INFO    MainThread:13803 [wandb_init.py:init():627] communicating run to backend with 30 second timeout
2022-06-07 20:29:55,690 INFO    MainThread:13803 [wandb_run.py:_on_init():1924] communicating current version
2022-06-07 20:29:55,898 INFO    MainThread:13803 [wandb_run.py:_on_init():1928] got version response 
2022-06-07 20:29:55,898 INFO    MainThread:13803 [wandb_init.py:init():662] starting run threads in backend
2022-06-07 20:30:00,919 INFO    MainThread:13803 [wandb_run.py:_console_start():1898] atexit reg
2022-06-07 20:30:00,920 INFO    MainThread:13803 [wandb_run.py:_redirect():1771] redirect: SettingsConsole.WRAP
2022-06-07 20:30:00,921 INFO    MainThread:13803 [wandb_run.py:_redirect():1808] Wrapping output streams.
2022-06-07 20:30:00,923 INFO    MainThread:13803 [wandb_run.py:_redirect():1832] Redirects installed.
2022-06-07 20:30:00,923 INFO    MainThread:13803 [wandb_init.py:init():688] run started, returning control to user process
2022-06-07 20:30:00,924 INFO    MainThread:13803 [wandb_watch.py:watch():47] Watching
2022-06-07 20:30:00,929 INFO    MainThread:13803 [wandb_init.py:_pause_backend():341] pausing backend
2022-06-07 20:30:00,929 INFO    MainThread:13803 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-06-07 20:30:43,807 INFO    MainThread:13803 [wandb_init.py:_resume_backend():350] resuming backend
2022-06-07 20:30:43,809 INFO    MainThread:13803 [wandb_init.py:_pause_backend():341] pausing backend
2022-06-07 20:30:43,809 INFO    MainThread:13803 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-06-07 20:31:50,447 INFO    MainThread:13803 [wandb_init.py:_resume_backend():350] resuming backend
2022-06-07 20:31:50,449 INFO    MainThread:13803 [wandb_init.py:_pause_backend():341] pausing backend
2022-06-07 20:31:50,454 INFO    MainThread:13803 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-06-07 20:32:51,773 INFO    MainThread:13803 [wandb_init.py:_resume_backend():350] resuming backend
2022-06-07 20:48:55,911 INFO    MainThread:13803 [wandb_init.py:_pause_backend():341] pausing backend
2022-06-07 20:48:55,911 INFO    MainThread:13803 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-06-07 20:49:33,140 INFO    MainThread:13803 [wandb_init.py:_resume_backend():350] resuming backend
2022-06-07 20:49:33,141 INFO    MainThread:13803 [wandb_init.py:_pause_backend():341] pausing backend
2022-06-07 20:49:33,142 INFO    MainThread:13803 [jupyter.py:save_ipynb():377] not saving jupyter notebook
